common/src/main/resources/parserFieldValidValues.txt

```diff
@@ -1,14 +1,14 @@
 LEI|String with only letters and numbers
 ULI|String with only letters and numbers
 LAR Record Identifier|Integer: 2
-Application Date|Integer or NA
+Application Date|Valid Date in format yyyyMMdd or NA
```

Comment:
@KennethGudel  are we sure these are the correct date formate regex i.e. yyyy vs YYYY